interpreter/host/spectest.ml

```diff
@@ -17,7 +17,8 @@ let global (GlobalType (t, _) as gt) =
   in Global.alloc gt v
 
 let table = Table.alloc (TableType ({min = 10l; max = Some 20l}, AnyFuncType))
-let memory = Memory.alloc (MemoryType {min = 1l; max = Some 2l})
+let memory = Memory.alloc (MemoryType ({min = 1l; max = Some 2l}, Unshared))
+let shared_memory = Memory.alloc (MemoryType ({min = 1l; max = Some 2l}, Shared))
```

Comment:
Nit: line break (above as well while you're here)

Reply:
Done. (which one do you mean above?)